packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -166,6 +166,14 @@ abstract class FlutterCommand extends Command<void> {
         valueHelp: 'x.y.z');
   }
 
+  void usesIsolateFilterOption({@required bool hide}) {
+    argParser.addOption('isolate-filter',
+      defaultsTo: null,
+      hide: hide,
+      help: 'Specify the isolate control. All isolates are targeted when this '
```

Comment:
Rephrase the first sentence for clarity. Provide context for when a developer would want to use this.